### ISSUE TITLE
No subpixel rendering

### ISSUE THREAD

**Issue by drujd** [giddie/poppler-cairo-backend#9]
It seems that in the "Update for Poppler 0.90.1" update, the "0003-Apply-subpixel-rendering-in-Cairo-Backend.patch" has been removed and subpixel rendering does not work in Okular anymore, so while the PDFs are rendered in Cairo, there is no subpixel rendering. Is there a reason for that?

**Comment by drujd** [comment]
Well, I tried the older version containing the subpixel rendering patch and it does not enable subpixel rendering either. Maybe some versioning mismatch? I am running 20.09.0. That being said, I tried rebasing the commits/patches to that version and there was no problem - no difference in the patches at all.

**Comment by giddie** [comment]
Yeah, honestly I'm not sure - the part I wrote myself is the Cairo backend for poppler-qt. I'm not familiar with Cairo itself, so I was relying on a patch found elsewhere to apply subpixel rendering to Poppler/Cairo. Unfortunately, as you say, that patch doesn't seem to work any more. I don't have a lot of time to look into it these days, but if you have the inclination, it would be awesome if you could do some searching to see if there's an updated patch to apply subpixel rendering. If it's reported to work in Evince, it should would in Okular using this patchset.

**Comment by drujd** [comment]
Unfortunately no subpixel rendering in Evince either... I will try to have a look when I have time.
